src/activities/content/ContentEntity.js

```diff
@@ -0,0 +1,13 @@
+import { Entity } from '../../es6/Entity';
+
+/**
+ * ContentEntity class representation of a d2l content entity.
+ */
+export class ContentEntity extends Entity {
```

Comment:
Should these be `ContentEntity` or `ContentActivity`? Or even `ContentActivityEntity`?

Reply:
I think it will be personal preference on what are team thinks is best. Assignments is called `AssignmentEntity` so I'd be fine with either `ContentEntity` or `ContentActivityEntity` to stay in the same ballpark as assignments.

Reply:
That makes sense, `ContentEntity` probably is fine then, and it's in the `activities/content` path anyway.